src/core_modules/capture-core/components/Pages/MainPage/EventWorkingListsInit/RunningMutationsHandler/WorkingListsInitRunningMutationsHandler.component.js

```diff
@@ -0,0 +1,14 @@
+// @flow
+import React from 'react';
+import { withLoadingIndicator } from '../../../../../HOC';
+import { WorkingListsInitHeader } from '../Header';
+import type { Props } from './WorkingListsInitRunningMutationsHandler.types';
+
+const EventWorkingListsSetupWithLoadingIndicator = withLoadingIndicator()(WorkingListsInitHeader);
+
+export const WorkingListsInitRunningMutationsHandler = ({ mutationInProgress, ...passOnProps }: Props) => (
+    <EventWorkingListsSetupWithLoadingIndicator
+        {...passOnProps}
+        ready={!mutationInProgress}
+    />
+);
```

Comment:
I could see that this exact same code could live in the `/EventWorkingListsInitConnectionStatusResolver.component.js` and we could completely delete this fie and keep the nesting more lean.